packages/thirdweb/src/wallets/in-app/core/wallet/types.ts

```diff
@@ -11,11 +11,13 @@ export type InAppWalletConnectionOptions = (
   | MultiStepAuthArgsType
   | SingleStepAuthArgsType
 ) & {
+  walletId: string;
```

Comment:
hmmm this is mandatory now? can we default to 'inApp' ? otherwise it'll break the API